### DISCUSSION TITLE
Filtering of activities by company

### DISCUSSION THREAD

**Discussion by Biermannek** [ghostfolio/ghostfolio#3608] [Q&A]
I'm almost certain it was possible in older versions to filter displayed records in "Portfolio - Activities" by holding  e.g. to display only records related to Apple. Was it removed? I found it useful, but can't found it.

**Comment by dtslvr** [comment]
Hi @Biermannek

This functionality has moved to the holdings detail dialog. You can now view the list of all activities by company (holding) there.

**Reply by Biermannek** [reply]
Hello @dtslvr ,

thank you for your answer. Problem is (at least for me), that in holding detail you see read only list only and you haven't available any actions for those activities. Unlike in "Portfolio - Activities", where you can Edit/Clone/Delete each of them. If this possibility was available in holding detail, then it would be great.

Just to show my use case - For me import of dividends is not 100% working and sometimes I have duplicities. With filtering I was able to download dividends and then check all activities of chosen company and see if there was some duplicity. If positive, then I just simply deleted it. But now I have to go to holding detail and if I find duplicity, I can't just delete it, but I have to switch to Portfolio - Activities, find duplicity there by date (which needs going through multiple pages) and then delete it.

Maybe these problems are caused only by some older records which I inserted manually long time ago, it's just a border case and I'm the only one who is using it :) But in fact I think it would be a nice feature if list of activities displayed inside holding detail had the same functionality as list of activities in Portfolio. Maybe even including import of dividends :)
